packages/marketplace/src/components/ui/web-component-config-modal/config-modal.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { Modal } from '@reapit/elements'
+import { WebComponentConfigModalInner } from './config-modal-inner'
+
+export const BOOK_VIEWING_CONSTANT = {
+  title: 'Book a Viewing Configuration',
+  subtext:
+    'Please use the following form to configure your diary settings for your ‘Book a Viewing’ widget on your website',
+}
+
+export const BOOK_VALUATION_CONSTANT = {
```

Comment:
I think when we define like this we know that CONSTANT, so you could remove suffix CONSTANT.